config/jobs/kubesphere/console/console-postsubmits.yaml

```diff
@@ -0,0 +1,51 @@
+postsubmits:
+  kubesphere/console:
+  - name: pull-console-build-image
+    labels:
+      preset-docker-sock: "true"
+      preset-docker-credential: "true"
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - command:
+        - "/bin/sh"
+        - "-c"
+        - "make image && make image-push"
+        image: kubesphere/build-tools:master-latest
+        name: ""
+        resources:
+          requests:
+            cpu: "1"
+            memory: 2Gi
+          securityContext:
+            privileged: true
+
+periodics:
+- interval: 24h
+  decorate: true
+  branches: master
+  extra_refs:
+  - org: kubesphere
+    repo: console
+    base_ref: master
+  name: kubesphere-console-periodic-build-image
+  labels:
+    preset-docker-sock: "true"
+    preset-docker-credential: "true"
+  spec:
+    containers:
+    - image: kubesphere/build-tools:master-latest
+      command:
+      - "/bin/sh"
+      - "-c"
+      - TAG=nightly-$(date "+%Y%m%d") make image && TAG=nightly-$(date "+%Y%m%d") make image-push
```

Comment:
Why don't we make TAG an env, make it globally across the whole build? Imagine `make image` happens at 23:59, and it took 10 mins to finish, so `make image-push` will have a different TAG with `make image`

Reply:
Nice catch. Done. `TAG=xxx` -> `export TAG`